_posts/en/newsletters/2024-02-28-newsletter.md

```diff
@@ -0,0 +1,303 @@
+    [extended documentation][cat vault readme] for the project notes
+    that you'd probably want to allow Alice to spend the money to a
+    different script in that case, and that the ideas behind his
+    construction allow for that but it isn't currently implemented for
+    simplicity.
+
+  These CAT-based vaults can be compared against the types of presigned
+  vaults we can create today without consensus changes and the
+  [BIP345][]-style `OP_VAULT` vaults that would provide the best-known
+  set of vault features if support for them was added in a soft fork.
+
+  <table>
+  <tr>
+    <th></th>
+    <th>Presigned</th>
+    <th markdown="span">BIP345 `OP_VAULT`</th>
+    <th markdown="span">`OP_CAT` with schnorr</th>
+  </tr>
+
+  <tr>
+    <th>Availability</th>
+    <td markdown="span">**Now**</td>
+    <td markdown="span">Requires soft fork of `OP_VAULT` and [OP_CTV][topic op_checktemplateverify]</td>
+    <td markdown="span">Requires soft fork of `OP_CAT`</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Last-minute address replacement attack</th>
+    <td markdown="span">Vulnerable</td>
+    <td markdown="span">**Not vulnerable**</td>
+    <td markdown="span">**Not vulnerable**</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Partial amount withdrawals</th>
+    <td markdown="span">Only if prearranged</td>
+    <td markdown="span">**Yes**</td>
+    <td markdown="span">No</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Static and non-interactive computable deposit addresses</th>
+    <td markdown="span">No</td>
+    <td markdown="span">**Yes**</td>
+    <td markdown="span">**Yes**</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Batched re-vaulting/quarantining for fee savings</th>
+    <td markdown="span">No</td>
+    <td markdown="span">**Yes**</td>
+    <td markdown="span">No</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Operational efficiency in best case, i.e. only legitimate spends<br>*(only very roughly estimated by Optech)*</th>
+    <td markdown="span">**2x size of regular single-sig**</td>
+    <td markdown="span">3x size of regular single-sig</td>
+    <td markdown="span">4x size of regular single-sig</td>
+  </tr>
+  </table>
+
+  The prototype had received a small amount of discussion and analysis
+  on the forum as of this writing.
+
+- **Sending and receiving ecash using LN and ZKCPs:** Anthony Towns
+  [posted][towns lnecash] to Delving Bitcoin about linking
+  "[ecash][topic ecash] mints to the lightning network without losing
+  ecash’s anonymity or adding any additional trust".  His proposal for
+  achieving that goal uses a zero-knowledge contingent payment
+  ([ZKCP][topic acc]) for sending payments to the user of an ecash mint
+  and a process of committing to a hash preimage for withdrawing ecash
+  funds to LN.
+
+  Calle, the lead developer of the [Cashu][] ecash implementation,
+  replied with some concerns but also support for the idea, a reference
```

Comment:
maybe link replied to https://delvingbitcoin.org/t/ecash-and-lightning-via-zkcp/586/2